bitbots_vision/docs/index.rst

```diff
@@ -4,48 +4,8 @@ Welcome to |project|'s documentation!
 Description
 -----------
 
-This is the vision ROS package of the Hamburg Bit-Bots.
-
-The vision is able to detect balls, lines, the field itself, the field boundary, goal posts, teammates, enemies and other obstacles.
-
-An earlier version of this pipeline is presented in our paper
-`An Open Source Vision Pipeline Approach for RoboCup Humanoid Soccer
-<https://robocup.informatik.uni-hamburg.de/wp-content/uploads/2019/06/vision_paper.pdf>`__.
-When you use this pipeline or parts of it, please cite it.
-
-::
-
-   @inproceedings{vision2019,
-   author={Fiedler, Niklas and Brandt, Hendrik and Gutsche, Jan and Vahl, Florian and Hagge, Jonas and Bestmann, Marc},
-   year={2019},
-   title={An Open Source Vision Pipeline Approach for RoboCup Humanoid Soccer},
-   booktitle={RoboCup 2019: Robot World Cup XXIII},
-   note = {Accepted},
-   organization={Springer}
-   }
-
-The Bit-Bots vision pipeline architecture is modular allowing easy implementation of new approaches resulting in a high grade of customizability.
-
-For ball detection, you can choose between multiple yolo implementations (or a legacy FCNN architecture).
-The goalpost detection also runs via yolo or a conventional detection method, which is also used for obstacle and robot detection.
-
-The whole system is embedded in the ROS environment and able to run on many devices including the Intel Neural Compute Stick 2 (and the Nvidia Jetson TX2, legacy) in our Wolfgang robots.
-
-In the context of the Hamburg Bit-Bots, the images are provided by a Basler industry grade ethernet camera.
-The camera drivers are not included in this package for licensing reasons, but can be auto launched.
-Every image source, that publishes a ``sensor_msgs/Image messages`` message is supported.
-
-The ROS topics and many other parameters are defined in the ``visionparams.yaml`` config file.
-All used parameters are also changeable during run-time using `ROS dynamic reconfigure <http://wiki.ros.org/dynamic_reconfigure>`_.
-For simulation usage, different parameters can be defined in the ``simparam.yaml`` file which overrides the default params.
-
-Neural network models are stored in the directory ``bitbots_vision/models``.
-These models are not part of this repository.
-
-For the field detection, which is needed for the field boundary and obstacle detection, the vision uses RGB lookup table provided by the ``colorpicker.py`` as in `Additional Scripts`_.
-These color lookup tables can be improved and converted to a pickle file for faster loading times using the ``color_lookup_table_tool``.
-The field color lookup table itself can be dynamically adapted in real-time using the dynamic color lookup table heuristic.
-Therefore the vision gets more resistant to natural light conditions.
+This is the vision ROS package of the Hamburg Bit-Bots. A description of the current, YOEO-based vision can be found
+:ref:`HERE <manual/yoeo_vision.rst>`. For the legacy vision, look :ref:`HERE <manual/legacy_vision.rst>`.
```

Comment:
Is either of you proficient enough in rst to judge whether these two refs will work?

Reply:
It may work. I would say, we just deploy it and see. If necessary, we fix it afterwards.